packages/aragon-api-react/README.md

```diff
@@ -0,0 +1,172 @@
+# aragonAPI for React
+
+This module lets you interact with aragonAPI using a [React Hook](https://reactjs.org/docs/hooks-intro.html) or, alternatively, with a [render prop](https://reactjs.org/docs/render-props.html). [`@aragon/api`](https://github.com/aragon/aragon.js/blob/master/docs/APP.md) is used under the hood, so being familiar with it can be useful.
+
+## Usage
+
+```jsx
+import { AragonApi, useAragonApi } from '@aragon/api-react'
+
+function App() {
+  const { api, appState } = useAragonApi()
+  const { count = 0 } = appState
+
+  return (
+    <div>
+      <div>{count}</div>
+      <button onClick={() => api.increment(1)}>
+        Increment
+      </button>
+    </div>
+  )
+}
+
+ReactDOM.render(
+  <AragonApi>
+    <App />
+  </AragonApi>,
+  document.getElementById('root')
+)
+```
+
+This is a simple example demonstrating how we can use aragonAPI for React to connect the app to its contract, fetch some data from its state (using `appState`), and trigger an action on it (with `api.increment(1)`). The full API is detailed below.
+
+## Documentation
+
+### &lt;AragonApi />
+
+Before using any Hook provided, you need to declare this component to connect the app. It is generally a good idea to do it near the top level of your React tree. It should only be declared once.
+
+It has an optional `reducer` prop, which lets you process the state coming from the background script. If not provided, the state is passed as is from the background script.
```

Comment:
Might be nice to link to https://hack.aragon.org/docs/aragonjs-guide-bg-scripts.html or https://github.com/aragon/aragon.js/blob/master/docs/BACKGROUND_SCRIPTS.md